cogs/tools.py

```diff
@@ -134,6 +138,62 @@ async def ask_question(self, ctx, conversation_mode="true", *, question: str):
 
         await ctx.send(embed=embed)
 
+    @commands.command()
+    async def charinfo(self, ctx, *, characters: str):
+        """Shows you information about a number of characters. Only up to 25 characters at a time."""
+        def to_string(c):
+            digit = f'{ord(c):x}'
+            name = unicodedata.name(c, 'Name not found.')
+            return f'`\\U{digit:>08}`: {name} - {c} \N{EM DASH} <http://www.fileformat.info/info/unicode/char/{digit}>'
+
+        msg = '\n'.join(map(to_string, characters))
+        if len(msg) > 2000:
+            return await ctx.send('Output too long to display.')
+        await ctx.send(msg)
+
+    @commands.command()
+    async def charinfo_uni(self, ctx: commands.Context, *, characters: str) -> None:
```

Comment:
Why is this the only type-hinted function? Either don't type hint at all, or type-hint everywhere.
Full type hinting will be added in PR #11 until that is merged it's not necessary.
Same goes for line 176

Reply:
i will remove the type hint, u edit it later.

Reply:
Yeah, we need to get type-hints and we also need to fill up metadata for the bot.
Also, we'll need to add docstrings for each function, class etc for better usability.

Reply:
@Theodore-Robinson we're Editing and adding the Type Hints and i am Trying to implement the docstrings everywhere